docs/edition_states.md

```diff
@@ -0,0 +1,64 @@
+# Edition States
+
+## Proposed State Diagram
+
+```mermaid
+flowchart TD
+    Pub[published]
+    Sup(superseded)
+    Dra(draft)
+    A2I[awaiting_2i]
+    AFC[awaiting_factcheck]
+    Sch[scheduled]
+
+    Dra -->|ready_for_2i| A2I
+    A2I -->|ready_for_factcheck| AFC
+    AFC -->|schedule| Sch
+    Sch -->|supersede| Sup
+    Sch -->|publish| Pub
+    AFC -->|publish| Pub
+    Pub -->|supersede| Sup
+```
+
+## Notes
+There are no backwards transitions in this model. If a user makes edits to a Published or Scheduled Edition, we 
```

Comment:
Am I right in saying that if a user makes an edit to an Edition in awaiting_i2 or awaiting_factcheck then the Edition remains in the same state?

Reply:
yes I believe that's what we agreed would work as a "minimum viable"